packages/web-components/src/design-tokens.ts

```diff
@@ -2003,78 +2051,94 @@ export const colorPaletteAnchorForeground2 = create<string>('colorPaletteAnchorF
  */
 export const colorPaletteAnchorBorderActive = create<string>('colorPaletteAnchorBorderActive').withDefault('#394146');
 
+// ----------------------------------------
+// Shadow
+// ----------------------------------------
+
 /**
  * @public
  */
-export const shadow2 = create<string>('shadow2').withDefault('0 0 2px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.14)');
+export const shadow2 = create<string>('shadow2').withDefault(
+  `0 0 2px ${colorNeutralShadowAmbient.$value}, 0 1px 2px ${colorNeutralShadowKey.$value}`,
```

Comment:
I'm not sure you need `$value` here, you should be able to reference the token itself.

Reply:
Will do. Does the `$` prefix mean private, or something else?

Reply:
No but in the use you have you're essentially just interpolating the value of another token so my thought is it should just be a direct reference...